packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -1095,6 +1196,13 @@ contract OPContractsManagerDeployer is OPContractsManagerBase {
         data = encodeETHLockboxInitializer(output, portals);
         upgradeToAndCall(output.opChainProxyAdmin, address(output.ethLockboxProxy), implementation.ethLockboxImpl, data);
```

Comment:
Should this `Initialize the ETHLockbox` check be behind `if (isDevFeatureEnabled(DevFeatures.OPTIMISM_PORTAL_INTEROP))`? Similar for this block:
```solidity
output.ethLockboxProxy =
  IETHLockbox(deployProxy(_input.l2ChainId, output.opChainProxyAdmin, _input.saltMixer, "ETHLockbox"));
```